common/docker/client_network.go

```diff
@@ -0,0 +1,64 @@
+package docker
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/docker/docker/api/types/network"
+
+	"pkg.world.dev/world-cli/common/logger"
+	"pkg.world.dev/world-cli/tea/style"
+)
+
+func (c *Client) createNetworkIfNotExists(networkName string) error {
+	ctx := context.Background()
```

Comment:
context should be passed down from the CLI command level; otherwise it wouldn't be very useful since you can't propagate the cancellation signal.

Reply:
adjusted